operator/controllers/installation_controller.go

```diff
@@ -457,6 +458,13 @@ func (r *InstallationReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		return ctrl.Result{}, fmt.Errorf("failed to ensure kotsadm CA configmap: %w", err)
 	}
 
+	if in.Status.State == ecv1beta1.InstallationStateInstalled {
```

Comment:
how can we ensure that this is the installation object that's tied to the upgrade job?